docs/rfcs/0006-site-wide-notification.md

```diff
@@ -0,0 +1,48 @@
+## Site Wide Notifications
+
+#### Abstract
+
+As a member of the marketing team, I'd like to be able to post a message that will appear on all pages of the site, for example to advertise a promotion.
+
+#### Requirements
+
+- Brief description of the plan for adding, editing and removing the site-wide notification (see options below)
+- CMS user can add, edit and remove banner message through Wagtail. Or, if that is difficult, admin user can add, edit and remove banner message through the django admin
```

Comment:
We probably want a few other things here:
- date range between which the content should be displayed
- ensure that only one site wide notification can be displayed at once (@pdpinch I'm guessing we want this to avoid overwhelming the user?)

Reply:
@rhysyngsun I agree. I suggest we don't spend to much time with some of these edge cases now.  scheduled publishing and restrict to one notification can be in a separate PRs